### PR TITLE
Add tests for sets command and for isolate select in Vp2RenderDelegate.

### PR DESCRIPTION
**Pull request by williamkrick**

### CODE REVIEW

**Comment by williamkrick** [comment]
I have one additional Maya change that is necessary to prevent the isolate select test from crashing, so do not merge this until that change is available.

File: test/lib/mayaUsd/render/vp2RenderDelegate/testVP2RenderDelegateIsolateSelect.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    """
+    Tests imaging using the Viewport 2.0 render delegate when using per-instance
+    inherited data on instances.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        # The test USD data is authored Z-up, so make sure Maya is configured
+        # that way too.
+        # cmds.upAxis(axis='z')
```
Comment (ppt-adsk) [inline test/lib/mayaUsd/render/vp2RenderDelegate/testVP2RenderDelegateIsolateSelect.py:48]: Commented out?

File: test/lib/ufe/testSetsCmd.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+# limitations under the License.
+#
+
+import fixturesUtils
+import mayaUtils
+import testUtils
+import ufeUtils
+import usdUtils
+
+import mayaUsd.ufe
```
Comment (ppt-adsk) [inline test/lib/ufe/testSetsCmd.py:25]: Don't think any of these are used.

File: test/lib/ufe/testSetsCmd.py [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+from pxr import Kind, Usd, UsdGeom, Vt
+
+from maya import cmds
+from maya import standalone
+
+import ufe
+
+import os
```
Comment (ppt-adsk) [inline test/lib/ufe/testSetsCmd.py:34]: Unused.

**Review by ppt-adsk — CHANGES_REQUESTED** [review]
testSetsCmd looks pretty good already, does need some undo / redo testing though.

File: test/lib/ufe/testSetsCmd.py [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+        # query set contents, remove from set, add to set
+        self.assertTrue(cmds.sets( usdCube, im=set1 )) # Test if Cube1 is in set1
+        self.assertFalse(cmds.sets( usdCube, im=set2 )) # Test if Cube1 is in set2
+        cmds.sets( usdCube, rm=set1 ) # Remove Cube1 from set1
+        self.assertFalse(cmds.sets( usdCube, im=set1 )) # Test if Cube1 is in set1
+        cmds.sets( usdCube, add=set1 ) # Add Cube1 to set1
+        self.assertTrue(cmds.sets( usdCube, im=set1 )) # Test if Cube1 is in set1
+
+        # Undo, Redo
```
Comment (ppt-adsk) [inline test/lib/ufe/testSetsCmd.py:90]: Looks like redo() never actually made it into the file :)

**Review by ppt-adsk — CHANGES_REQUESTED** [review]

**Review by ppt-adsk — APPROVED** [review]

File: test/lib/mayaUsd/render/vp2RenderDelegate/testVP2RenderDelegateIsolateSelect.py [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+        cmds.move(-4, -24, 0, "persp")
+        cmds.rotate(90, 0, 0, "persp")
+
+        globalSelection = ufe.GlobalSelection.get()
+        globalSelection.clear()
+        self.assertSnapshotClose('unselected.png')
+
+        # Turn on isolate select for cube
+        cmds.select(usdCube)
+        cmds.isolateSelect("modelPanel4", state=1)
```
Comment (JGamache-autodesk) [inline test/lib/mayaUsd/render/vp2RenderDelegate/testVP2RenderDelegateIsolateSelect.py:89]: Can this function use the result of `mayaUtils.activeModelPanel()` as first parameter? Would it make the test more robust?

**Review by JGamache-autodesk — APPROVED** [review]

**Pull request merged**